autoload/tsuquyomi/config.vim

```diff
@@ -21,11 +19,11 @@ let s:tss_version = {'is_valid': 0, 'out': '???'}
 function! tsuquyomi#config#preconfig()
 
   if !exists('g:tsuquyomi_is_available')
-    if !s:P.is_available()
-      " 1. vimproc installation check
+    if !has('patch-8.0.0')
```

Comment:
my thought was vim could be compiled without lambda and jobs support. If not the case I'm ok